index.html

```diff
@@ -409,6 +409,8 @@ <h2>Firing events using the <code>PointerEvent</code> interface</h2>
                 <p>If the event is {{GlobalEventHandlers/pointerdown}}, the associated device is a direct manipulation device, and the target is an {{Element}},
                    then <a>set pointer capture</a> for this <code>pointerId</code> to the target element as described in <a>implicit pointer capture</a>.
 
+                <p>Run <a href="html.spec.whatwg.org/#popover-light-dismiss">light dismiss open popovers</a> given the event and target.</p>
```

Comment:
Since this is now crossing specs, the interface could be simplified to just giving the event, and HTML can get `event.target`, effectively.

Reply:
In fact the algorithm in HTML already gets event target. @josepharhar do you think it makes sense to reduce it to just one argument?